src/test/java/gov/usgs/wma/waterdata/collections/CollectionsControllerIT.java

```diff
@@ -93,6 +99,44 @@ public void collectionNotFoundTest() {
 		assertNull(rtn.getBody());
 	}
 
+	@Test
+	public void collectionFeaturesTest() {
+		for (String collectionId : COLLECTION_IDS) {
```

Comment:
These tests are very hard to follow. Smells like a lot of hard to maintain java code when a simple hamcrest based test would do. see the MonitoringLocationControllerIT.foundTest()